pages/company/index.tsx

```diff
@@ -84,15 +100,12 @@ const Index = ({
     );
   }, []);
 
+  const Microcopy = createMicrocopyComponent(microcopy);
```

Comment:
Certainly doesn't _have_ to be this way, I just thought it was the cleanest, especially since there will potentially be very many `Microcopy` components, wanted to eliminate any repetition.

Potential alternatives include: 
- Pass the array of `Entry<TypeMicrocopyFields>` to each `Microcopy` component, along with the id
- Pass the appropriate `Entry<TypeMicrocopyFields>` directly to the `Microcopy` component, either with a `.find` in each component or somewhere else